internal/api/grpc/admin/integration_test/iam_member_test.go

```diff
@@ -180,9 +184,12 @@ func TestServer_AddIAMMember(t *testing.T) {
 			if tt.wantErr {
 				assert.Error(t, err)
 				return
+			} else {
+				if !assert.NoError(t, err) {
+					return
+				}
+				integration.AssertDetails(t, tt.want, got)
 			}
```

Comment:
There is a `return` statement in the above condition. The `else` block is unneeded.

```suggestion
			}
            if !assert.NoError(t, err) {
                return
            }
            integration.AssertDetails(t, tt.want, got)
```

Reply:
Comment applies to all cases in this PR where

```
if <condition> {
    <check>
    return
} else {
    ....
}
```